backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/age/AgeAttributesService.java

```diff
@@ -94,4 +84,67 @@ public int computeDefaultAge()
 
 		return attributesRepo.retrieveAttributeValues(age);
 	}
+
+	public List<Object> getSuitableValues(final Set<BPartnerId> bPartnerIds, final Set<ProductId> productIds, final Object attributeValue)
```

Comment:
LE: maybe reaname the method toExtractMatchingValues? might be misleading..i'm not sure what the method does..but "get" makes it sound more trivial than it is..